source/features/unfinished-comments.tsx

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let documentTitle: string | null = null;
+
+async function updateDocumentTitle(): Promise<void> {
+	if (document.visibilityState === 'hidden') {
+		if (select.all('textarea').some(textarea => textarea.value.length > 0 && (textarea.offsetWidth > 0 || textarea.offsetHeight > 0))) {
```

Comment:
We only ever need to check one dimension:
```suggestion
		if (select.all('textarea').some(textarea => textarea.value.length > 0 && textarea.offsetWidth > 0) {
```